externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpOps.td

```diff
@@ -158,6 +158,26 @@ def Tcp_DivFOp : Tcp_BinaryElementwiseOp<"divf", [SameOperandsAndResultElementTy
   let assemblyFormat = "$in1 `,` $in2 attr-dict `:` type($in1) `,` type($in2) `->` type($out)";
 }
 
+def Tcp_ConstOp : Tcp_Op<"const", [ConstantLike, Pure]> {
+  let summary = "Constant op";
+
+  let description = [{
+    An op that produces constant data that are used in other ops.
```

Comment:
Any constraints on the data types?

Reply:
`tcp.const` op supports `Tcp_Tensor` type (floating point, signless integers, complex) for the result, however when lowering from TorchToTcp, for now we only support floating point cases (captured a TODO for future support of integer use-cases). Will clarify in the op's description.